app/Jobs/FetchCollectionFloorPrice.php

```diff
@@ -100,4 +103,18 @@ public function retryUntil(): DateTime
     {
         return now()->addMinutes(30);
     }
+
+    /**
+     * Get the middleware the job should pass through.
+     *
+     * @return array<int, object>
+     */
+    public function middleware(): array
+    {
+        if (Config::get('dashbrd.web3_providers.'.self::class) === 'opensea') {
+            return [new RateLimited(Service::Opensea)];
```

Comment:
tested this locally with the opensea request commented so it would only queue the job and update the collection's `opensea_slug_last_fetched_at` value. This caused no fails on #168 (838/838 collections updated), but on this PR it only updated around 10% (84/838)